examples/mouse_drag_callback.py

```diff
@@ -0,0 +1,45 @@
+"""
+Example updating the status bar with line profile info while dragging
+lines around in a shapes layer.
+"""
+
+from skimage import data
+from skimage import measure
+import numpy as np
+import napari
+
+
+def profile_lines(image, shape_layer):
+    profile_data = []
+    for line in shape_layer.data:
+        profile_data.append(
+            measure.profile_line(image, line[0], line[1]).mean()
+        )
+    msg = ('profile means: ['
+            + ', '.join([f'{d:.2f}' for d in profile_data])
+            + ']')
+    shape_layer.status = msg
+
+
+
```

Comment:
not sure why black is failing as I thought we weren't testing the examples anymore, regardless maybe extra line here